Source/Core/Core/NetPlayServer.cpp

```diff
@@ -2067,9 +2068,17 @@ bool NetPlayServer::SyncCodes()
   }
   // Sync Gecko Codes
   {
+    std::vector<Gecko::GeckoCode> s_codes = Gecko::LoadCodes(globalIni, localIni);
```

Comment:
This variable isn't static.

```suggestion
    std::vector<Gecko::GeckoCode> codes = Gecko::LoadCodes(globalIni, localIni);
```